sw/device/silicon_creator/lib/base/rom_mmio.h

```diff
@@ -0,0 +1,71 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_BASE_ROM_MMIO_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_BASE_ROM_MMIO_H_
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include "sw/device/lib/base/mmio.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @file
+ * @brief ROM Memory-mapped IO functions, for volatile access.
+ *
+ * Memory-mapped IO functions, which map to volatile accesses. Use this module
+ * for register operations in mask ROM and ROM Extension production libraries.
+ */
+
+/**
+ * All MMIO functions return their results using return values, rather than out-
+ * parameters. Where the return types are non-void, it is prudent to ensure
+ * these results are used, or explicitly discarded (in the case of a volatile
+ * read that is needed for side effects only).
+ */
+#define ROM_MMIO_WARN_UNUSED_RESULT __attribute__((warn_unused_result))
+
+/**
+ * Reads an aligned uint8_t from MMIO `addr`.
```

Comment:
It says uint8 here but the type is uint32. Is this intentional...?

(Aligned is also redundant in this position.)

Reply:
Fixed.